### PR TITLE
[TASK] Update TYPO3 repository URL

### PR DESCRIPTION
**Pull request by alexander-nitsche**
See https://github.com/TYPO3-Documentation/T3DocTeam/issues/161 for details.

### CODE REVIEW

File: Documentation/Appendix/Windows/GitForWindows.rst [hunk 1]
```diff
@@ -127,7 +127,7 @@ Push to Gerrit
 
 In the Git Bash window, enter the following commands to set that you push to Gerrit instead of the TYPO3 repository directly.
 
-git config --global url."ssh://<username>@review.typo3.org:29418".pushInsteadOf git://git.typo3.org
+git config --global url."ssh://<username>@review.typo3.org:29418/Packages/TYPO3.CMS.git"
```
Comment (susannemoog) [inline Documentation/Appendix/Windows/GitForWindows.rst:130]: Does this set just the PUSH Url or does it set both pull and push? With pushInsteadOf that was only relevant to push, but I think to change the push url only the command itself needs to be adjusted.

Comment (susannemoog) [reply Documentation/Appendix/Windows/GitForWindows.rst:130]: (additionally: restricting this globally to TYPO3.CMS might not be a good idea, as there have been - and maybe still are - other projects using Gerrit)

**Review by susannemoog — CHANGES_REQUESTED** [review]
see single comments

File: Documentation/Appendix/Windows/GitForWindows.rst (hunk 1 above)
Comment (alexander-nitsche) [reply Documentation/Appendix/Windows/GitForWindows.rst:130]: I guess i just aligned it with https://github.com/TYPO3-Documentation/TYPO3CMS-Guide-ContributionWorkflow/pull/205/files#diff-ab8c729c7a5a0fdf344ff1e9c4752b3b50e59a537628cd54b5cd173d17e5739eR140 . Did not test it really.

Comment (susannemoog) [reply Documentation/Appendix/Windows/GitForWindows.rst:130]: check the link you posted: there is `git config remote.origin.pushurl` for gerrit - I guess this is what we need here.

Comment (alexander-nitsche) [reply Documentation/Appendix/Windows/GitForWindows.rst:130]: hi susi, what if you would update the commit? i do not feel comfortable enough in the gerrit area ..

**Review by susannemoog — APPROVED** [review]

**Pull request merged**